src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+  padding: 0 10px 0 8px;
+  box-sizing: border-box;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.big input {
+  margin: 20px 0;
+  height: 68px;
+  padding-left: 62px;
+}
+
+.big input::placeholder {
+  font-size: 15px;
+  font-weight: 400;
+}
+
+.big,
+.small {
+  position: relative;
+}
+
+.big img {
+  position: absolute;
+  top: 3%;
+  left: 26px;
+}
+
+.small input {
+  height: 40px;
+  padding-left: 33px;
+}
+
+.small input::placeholder {
+  font-size: 13px;
+  font-weight: 400;
```

Move it to small__input, not only for placeholder, and font-size should be 14px

```suggestion
  font-size: 14px;
  font-weight: 400;
```